tests/zkevm/test_worst_compute.py

```diff
@@ -104,3 +117,207 @@ def test_worst_keccak(
+        env=env,
+        pre=pre,
+        post={},
+        blocks=[Block(txs=[tx])],
+    )
+
+
+@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize(
+    "gas_limit",
+    [
+        36_000_000,
+    ],
+)
+def test_jumpdest_only(
+    blockchain_test: BlockchainTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    gas_limit: int,
+):
+    """Test with only JUMPDEST opcodes"""
+    gsc = fork.gas_costs()
+    iteration_gas_cost = gsc.G_JUMPDEST
+
+    # Use Op.JUMPDEST instead of raw bytes
+    opcodes_pattern = Op.JUMPDEST
+
+    run_jump_analysis_test(
+        blockchain_test=blockchain_test,
+        pre=pre,
+        fork=fork,
+        gas_limit=gas_limit,
+        opcodes_pattern=opcodes_pattern,
+        iteration_gas_cost=iteration_gas_cost,
+        description="JUMPDEST only",
+    )
+
+
+@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize(
+    "gas_limit",
+    [
+        36_000_000,
+    ],
+)
+def test_jumpdest_jump(
+    blockchain_test: BlockchainTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    gas_limit: int,
+):
+    """Test with JUMPDEST and JUMP opcodes"""
+    gsc = fork.gas_costs()
+    iteration_gas_cost = gsc.G_JUMPDEST + gsc.G_MID
+
+    # Use Op.JUMPDEST + Op.JUMP instead of raw bytes
+    opcodes_pattern = Op.JUMPDEST + Op.JUMP
+
+    run_jump_analysis_test(
+        blockchain_test=blockchain_test,
+        pre=pre,
+        fork=fork,
+        gas_limit=gas_limit,
+        opcodes_pattern=opcodes_pattern,
+        iteration_gas_cost=iteration_gas_cost,
+        description="JUMPDEST + JUMP",
+    )
+
+
+@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize(
+    "gas_limit",
+    [
+        36_000_000,
+    ],
+)
+def test_jumpdest_push0_pop_jump(
+    blockchain_test: BlockchainTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    gas_limit: int,
+):
+    """Test with JUMPDEST, PUSH0, POP, and JUMP opcodes"""
+    gsc = fork.gas_costs()
+    iteration_gas_cost = gsc.G_JUMPDEST + gsc.G_VERY_LOW + gsc.G_BASE + gsc.G_MID
+
+    # Use Op.JUMPDEST + Op.PUSH0 + Op.POP + Op.JUMP instead of raw bytes
+    opcodes_pattern = Op.JUMPDEST + Op.PUSH0 + Op.POP + Op.JUMP
+
+    run_jump_analysis_test(
+        blockchain_test=blockchain_test,
+        pre=pre,
+        fork=fork,
+        gas_limit=gas_limit,
+        opcodes_pattern=opcodes_pattern,
+        iteration_gas_cost=iteration_gas_cost,
+        description="JUMPDEST + PUSH0 + POP + JUMP",
+    )
+
+
+@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize(
+    "gas_limit",
+    [36_000_000],
+)
+def test_push0_pop_no_jump(
+    blockchain_test: BlockchainTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    gas_limit: int,
+):
+    """Test with just PUSH0 and POP opcodes"""
+    gsc = fork.gas_costs()
+    iteration_gas_cost = gsc.G_VERY_LOW + gsc.G_BASE
+
+    # Use Op.PUSH0 + Op.POP instead of raw bytes
+    opcodes_pattern = Op.PUSH0 + Op.POP
```

Comment:
This sounds fine to me. The reason I did it in my gist was to compare the cycles of these simple opcodes with the JUMP/JUMPDEST ones. Kind of as a baseline.